src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -343,7 +350,88 @@ public void Stop()
             }
         }
 
+        /* W3C support functionality (see https://w3c.github.io/trace-context) */
+
+        /// <summary>
+        /// Holds the W3C 'tracestate' header.   This is typically used by logging systems
+        /// to store vendor-specific information that must flow.
+        /// Logically it is just a kind of baggage (if flows just like baggage), but because
+        /// it is expected to be special cased (it has its own HTTP header), it is more 
+        /// convenient/efficient if it is not lumped in with other baggage.   
+        /// </summary>
+        public string TraceState
```

Comment:
Spec uses tracestate as a word. Please do not upper case `State`. See discussion https://github.com/census-instrumentation/opencensus-specs/pull/171

Reply:
Other http headers/consts that I know of in Microsoft/System.Net are written with .net naming conventions.

Reply:
`Tracestate` follows naming convention if considered to be a single word.